README.md

```diff
@@ -5,13 +5,7 @@ This is a quickstart Rails application for OpenShift v3 that you can use as a st
 
 If you'd like to install it, follow [these directions](https://github.com/openshift/rails-ex/blob/master/README.md#installation).  
 
-In order to access the example blog application, you have to remove the
-`public/index.html` which serves as the welcome page. Another option is to make a
-request directly to `/articles` which will give you access to the blog.
-
-The username/pw used for authentication in this application are openshift/secret.
-
-The following steps assume that you have access to an OpenShift deployment; you must have an OpenShift deployment that you have access to in order to deploy this app.
+The steps in this document assume that you have access to an OpenShift deployment that you can deploy applications on.
```

Comment:
How about we use "OpenShift cluster" instead of "OpenShift deployment", specially when the verb "deploy" comes 4 words later?